_posts/2016-10-10-episerver-developer-meetup-solita.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Episerver developer meetup at Solita
+author: Rinorragi
+date: 2016-10-10 14:00:00 +0200
+excerpt: What happened in the second Episerver developer meetup held in Helsinki
+categories: 
+- Episerver
+tags: 
+- Episerver 
+- DOTNET 
+- security
+---
+Solita organized a meetup for Finnish Episerver developers in Helsinki at 29.9. Meetup consisted of food, beer and presentations. It was second of a kind and as fun and cozy as its successor. This time we had a bit more time in the end for the networking than last time!
+
+## Commerce
+
+Johan Hedberg from Commerce development team visited us and gave us a presentation about how to use Commerce to your daily household finance governance. He showed us how to invent your own currency in Commerce, how to give users credit to be used inside the Commerce and how to use the credit for shopping. In addition to that he showed how to use promotion engine in household to give stuff cheaper to household members when they do laundry or clean toilets. It was a very nice presentation. There not much slides but Johan promised that the example will be available in some format at some day. Looking forward for that! 
+
+## Performance
+
+![Tools](/img/episerver-meetup/tommi.jpg)
+
+Tommi Raunio from Solita had a presentation about performance. He first talked about what is performance and why does it matter and then exploded the topic into three different segments: design & architecture, implement and measure. The slides can be found [here](http://www.slideshare.net/Solita_Oy/episerver-dev-meetup-performance-in-episerver-solutions-tommi-raunio-solita)
+. 
+
+## DevOps
+
+![Tools](/img/episerver-meetup/joona.jpg)
+
+I also had a presentation of my own about DevOps with Episercer. I went through few topics in my presentation. How to help new guys in installing development environments. Then I talked about how to script server installations with PowerShell. Then I went a bit deeper and talked about how to build a build pipeline and finally I talked about something about NuGets. The [Slides](http://www.slideshare.net/Solita_Oy/epihelsinki-episerverdevops-joonaimmonen-solita).
```

Comment:
How to help new guys in installing development environments => Tästä ei taida käydä ilmi, että kyse oli nimenomaan skriptastusta asennuksesta (Esim. ei siitä että miten neuvotaan kehittäjiä paremmin)